d3a_api_client/__init__.py

```diff
@@ -0,0 +1,16 @@
+import logging
+import sys
+
+import gsy_e_sdk
+
+logger = logging.getLogger(__name__)
+console_handler = logging.StreamHandler()
+console_handler.setFormatter(logging.Formatter("%(levelname)s: %(message)s"))
+logger.addHandler(console_handler)
+
+logger.warning(
+    "The d3a_api_client module name will be deprecated soon. Please use gsy_e_sdk instead.")
```

Comment:
Minor, but please use `The d3a_api_client package name will be deprecated soon.`